asset/asset.go

```diff
@@ -658,6 +686,14 @@ type Asset struct {
 	// its unique ID within the Taproot Asset protocol.
 	Genesis
 
+	// Type uniquely identifies the type of Taproot asset.
+	ExplicitType Type
+
+	// ID is an asset's unique identifier within the Taproot Asset protocol.
```

Comment:
nit: Godoc comment name doesn't match struct member name (here and for `ExplicitType`).